pxf/pxf-api/src/main/java/org/apache/hawq/pxf/api/FilterParser.java

```diff
@@ -396,4 +387,12 @@ private Operation reverseOp(Operation operation) {
         operatorTranslationMap.put(8, Operation.HDOP_LIKE);
         return operatorTranslationMap;
     }
+
+    static private Map<Integer, LogicalOperation> initLogicalOperatorTransMap() {
+        Map<Integer, LogicalOperation> integerLogicalOperationMap = new HashMap<>();
```

Comment:
array will be a better structure for faster access of lookup of integer from 0..N to an object.